code/modules/client/preferences/preferences.dm

```diff
@@ -131,7 +131,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	if(pref_load == PREFERENCE_LOAD_SUCCESS || pref_load == PREFERENCE_LOAD_NO_DATA)
 		log_preferences("[parent?.ckey]: Player preferences loaded and applied.")
 		if("6030fe461e610e2be3a2c3e75c06067e" in purchased_gear) //MD5 hash of, "extra character slot"
-			max_save_slots += 1
+			max_save_slots += 2
```

Comment:
Needs to be reworked a bit. Maybe a modifier on the hash to determine the number of purchases?